docs/analytics/screen-tracking.md

```diff
@@ -50,7 +51,7 @@ Navigation.events().registerComponentDidAppearListener(async ({ componentName, c
   if (componentType === 'Component') {
     await analytics().logScreenView({
       screen_name: componentName,
-      screen_class: componentName,
+      screen_class: componentName
```

Comment:
this is a matter of taste, with the community (and the default / our default prettier config) leaning towards trailing commas these days: https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker

```suggestion
      screen_class: componentName,
```